src/index.js

```diff
@@ -5,7 +5,17 @@
  */
 
 const hammingDistance = (a, b) => {
-    
+  if (a.length !== b.length){
+    const error = new Error('No puedes comparar dos string de distinto tamaño')
+      throw error;
+    }
+  let counter = 0;
+  for( let index = 0; index < a.length; index++){
```

Comment:
Recuerda que los strings son iterables, entonces funciona `for(i in a)` e `i` tiene el número de indice de la posición.